From 7ba1de909c587b6773a39ba9f6f9890987f35b24 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 24 Jul 2018 11:30:53 +0800
Subject: [PATCH] libtevent: fix musl libc compile error

libldb depend on libtevent,  build libldb with musl libc,
will report error:
tevent.h:1440:8: error: unknown type name 'pid_t'; did you mean 'div_t'?

add missing headers to fix it.

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 tevent.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tevent.h b/tevent.h
index aa6fe0d..2572696 100644
--- a/tevent.h
+++ b/tevent.h
@@ -32,6 +32,8 @@
 #include <talloc.h>
 #include <sys/time.h>
 #include <stdbool.h>
+#include <sys/stat.h>
+#include <sys/types.h>
 
 struct tevent_context;
 struct tevent_ops;
-- 
2.7.4

